CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Release report: TBD
 
 ### Added
 
+- Add new cases for the sync-agent-groups-get endpoint in wazuh_db ([#3959](https://github.com/wazuh/wazuh-qa/pull/3959))
```

Comment:
It's very generic. Be specific in that a case is added, and an output is changed.

Reply:
Fixed in commit [d4c2c6f](https://github.com/wazuh/wazuh-qa/pull/3959/commits/d4c2c6f26f42d7e12ee3049e3a07922d3025af46)